System:
  model: os1-64 
NodeGen:    
    lidar2body: [0.997, 0, 1.605, 0.0, 0.0, 1.0, 0.0] #64
    offset_x: -0.5 #-1.13  
    offset_y: 0.5 #W1.25
    # lidar2body: [0.997, 0, 1.605, 0.0, 0.0, 0.0, 1.0] #128
    # lidar2body : [0.970, 0, 1.560, 0, 0, 0, 1.0] # KETI
legoloam:
    rotation: [0, 0, -1,
              -1, 0, 0,
              0, 1, 0]
PATH:    
    lane_shp_path: /home/jsg/AutoCar/Material/[0] HdMap/2_LANE_SIDE_KAIST_Densified/40cm/2_LANE_SIDE_KAIST_40.shp
    lane_shp_layer: 2_LANE_SIDE_KAIST_40
    pole_shp_path: /home/jsg/current/map_ws/shapefile/pole_merged/pole_merged.shp
    pole_shp_layer: pole_merged
    # plane_shp_path: /media/jsg/Samsung_T5/data_1018/hdmap/Planar.shp
    # plane_shp_layer: Planar
    plane_shp_path: /media/jsg/T7/PointCloudMap/hdmap/planar/Planar_ver01.shp
    plane_shp_layer: Planar_ver01
ground:
  sensor_height: 1.7
  removal_height: 1.7
  num_iter: 3
  num_lpr: 100
  th_seeds: 1.2 #1.2
  th_dist: 0.5  #0.3
road:
  max_intensity : 1000.0
  min_intensity : 700.0
pole:
  x_clipped: 30.0 
  y_clipped: 15.0 
  z_axis_min: -1.5
  z_axis_max:  7.0
  cluster_size_min: 3 
  cluster_size_max: 100
  height_pole: 0.5
  height_max: 10.0 
  height_min: 2.0
  voxel_size: 0.2
  pole_radius: 0.2
  ec_tolerance: 0.3
  height_trunk: 1.0
  ec_tolerance_removed: 0.5

planar:
  z_axis_min: 7.0
  z_axis_max: 30.0
  cluster_size_min: 500
  cluster_size_max: 700000
  height_plane: 1.0
  height_max: 10.0 
  height_min: 1.0
  distance_plane: 3.0
  num_normal_kd: 100
  normal_weight: 0.5
  voxel_size: 0.2
  # region growing 
  rg_num_ksearch: 10 #50
  rg_mincluster: 50 # 1000 100??
  rg_numneighbor: 30 # 150  
  rg_smoothness: 3.0 
  rg_curvature: 1.0
  rs_threshold: 0.2 
  rs_threads: 10 
  rs_tolerance: 0.5 
  rs_min: 20
  rs_max: 100000
  length_threshold: 6.0

mcl:
    node_dist: 0.1
    particle_min: 100
    particle_max: 100
    covariance_odom: [0.02,0.02,0.02,0.002,0.02,0.005]  #rot-rot, trans-rot, trans-trans, rot-trans, x, y
    pole_max: 3
    static_velocity: 0.6
    planar_max: 1
    planar_cov: 1.0
    theta_noise: -0.0

sort:
    max_age: 3
    min_hits: 1
